hexa/core/filters.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.admin import SimpleListFilter
+from django.utils.translation import gettext_lazy as _
+
+
+class SoftDeleteFilter(SimpleListFilter):
+    title = "is deleted"
+    parameter_name = "is_deleted"
+
+    def lookups(self, request, model_admin):
+        return (
+            ("true", _("Soft deleted")),
+            ("false", _("Not Deleted")),
+        )
+
+    def queryset(self, request, queryset):
+        if not self.value():
+            return queryset
+
+        value = {"true": False, "false": True, "all": "all"}[self.value() or "false"]
```

Comment:
In which case is the `"all": "all"`  called ? What happens when you filter with deleted_at= "all" ?

Reply:
I forgot to remove that entry, it should not be there.